domain/consensus/model/utxo_diff_reversal_data.go

```diff
@@ -0,0 +1,9 @@
+package model
+
+import "github.com/kaspanet/kaspad/domain/consensus/model/externalapi"
+
+// UTXODiffReversalData is used by ConsensusStateManager to reverse the UTXODiffs during a re-org
+type UTXODiffReversalData struct {
+	SelectedParentHash     *externalapi.DomainHash
+	SelectedParentUTXODiff externalapi.UTXODiff
```

Comment:
Looks like this is not that easy - because it's in model, not in consensusStateManager (and it's not possible to move to consensusStateManager, because it is returned/passed to methods that are in model)